Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.11.0-alpine
+LABEL authors="serhiik"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /cinema_service
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
```

Comment:
It's generally a good practice to copy only the necessary files for installing dependencies. Consider using a multi-stage build or copying only the `requirements.txt` file initially to leverage Docker's caching mechanism more effectively.